lib/util/tXml.js

```diff
@@ -874,9 +874,9 @@ shaka.util.TXml = class {
    * @param {boolean=} doChildren
    * @return {!Element}
    */
-  static txmlNodeToDomElement(node, doParents = true, doChildren = true) {
+  static txmlNodeToDomElement(node, doParents = false, doChildren = true) {
```

Comment:
Since this isn't a publicly exported interface, and the one place this is called `doParents` isn't set manually, we can probably just remove the `doParents` and `doChildren` variables entirely. And just remove the `doParents` logic, and make the `doChildren` logic no longer be conditional.